kitsune/search/v2/signals/questions.py

```diff
@@ -0,0 +1,64 @@
+from django.db.models.signals import post_save, post_delete, m2m_changed
+from django.dispatch import receiver
+from django.conf import settings
+from kitsune.search.v2.es7_utils import (
+    index_object,
+    delete_object,
+    index_objects_bulk,
+    remove_from_field,
+)
+from kitsune.questions.models import Question, QuestionVote, Answer, AnswerVote
+from taggit.models import Tag
+
+
+@receiver(post_save, sender=Question)
+@receiver(m2m_changed, sender=Question.tags.through)
+def handle_question_save(instance, **kwargs):
+    if getattr(kwargs, "action", "").startswith("pre_"):
```

Comment:
why are we checking for `pre` signals here?

Reply:
`m2m_changed` sends a `pre_foobar` signal followed by a `post_foobar` signal (with the changes now made in DB). We want the latter, and I can't see a way to filter them out in the `@receiver` decorator - we have to subscribe to the whole lot.